dotnet/README.md

```diff
@@ -1,3 +1,6 @@
 [![Build Status](https://secure.travis-ci.org/cucumber/gherkin-dotnet.svg)](http://travis-ci.org/cucumber/gherkin-dotnet)
+[![Build status](https://ci.appveyor.com/api/projects/status/ka74of1ptpdfr6uv?svg=true)](https://ci.appveyor.com/project/SabotageAndi/gherkin)
```

Comment:
I never wanted that the stuff run in my AppVeyor account if this change is added. 
As noted in the PR- text, this Urls have to be changed to the real ones.

So when there is already an AppVeyor account, you only have to add the project.